### PR TITLE
Remove style conflicts

### PR DESCRIPTION
**Pull request by syntapy**
*NOTE: This is a dependency of [upgrade tailwind css](https://github.com/civiform/civiform/pull/3438) and  [replace Styles.java calls with style literals](https://github.com/civiform/civiform/pull/2630), i.e. this PR needs to be merged before either of those are merged*

### Description

Needed to fix broken styles in [replace Styles.java calls with style literals](https://github.com/civiform/civiform/pull/2630)

Some of our elements have conflicting tailwind styles

Which one ends up being shown depends on their order in the tailwind.css file

The order of styles in tailwind.css depends on the order they were found during extraction

Before it just so happened that when the Styles.java was parsed, conflicting style classes didn't have an effect

But when style classes are scattered throughout the UI code, it can cause conflicts

Please explain the changes you made here.

## Release notes:

Make UI element style selection 

### Checklist

Note: Rebase the style literals PR on top of this one and make sure no styles are broken to test this one out 
  - keep in mind the style literals PR needs to be rebased on top of the [tailwind upgrade PR](https://github.com/civiform/civiform/pull/3438), so rebase the tailwind upgrade PR off of this first
 
- [x] Fix conflicting border transparency / non-transparency in REVIEW button
- [x] Fix conflicting styles in LanguageSelector radio field borders (presumably thats the cause of that broken style)

### CODE REVIEW

**Comment by syntapy** [comment]
[This](https://github.com/civiform/civiform/blob/main/server/app/views/LanguageSelector.java#L96-L108) might be the location of a style conflict for the LanguageSelector radio button

File: server/app/views/style/StyleUtils.java [hunk 1]
```diff
@@ -113,4 +113,13 @@ public static String responsive2XLarge(ImmutableList<String> styles) {
   public static String joinStyles(String... styles) {
     return String.join(" ", styles);
   }
+
+  public static String removeStyles(String style, String... stylesToRemove) {
+    return Stream.of(stylesToRemove)
+        .reduce(
+            style,
+            (acc, styleToRemove) -> acc.replace(styleToRemove, ""),
+            (acc1, acc2) -> acc1 + acc2)
```
Comment (nbeloglazov) [inline server/app/views/style/StyleUtils.java:122]: I don't think you need to pass the last function. Having only accumulator function is enough.

**Review by nbeloglazov — APPROVED** [review]

File: server/app/views/style/StyleUtils.java (hunk 1 above)
Comment (syntapy) [reply server/app/views/style/StyleUtils.java]: Done. Yeah that code was actually written by Github Copilot

**Review by nbeloglazov — APPROVED** [review]

**Comment by syntapy** [comment]
There might be some more style conflicts breaking styles what with recent changes.

**Comment by nbeloglazov** [comment]
@syntapy I suggest you submit your current PR. If you find more conflicts in future - you can open another PR. Plus it will be easier for reviewers as we won't need to keep in our head what's new and what's old.

**Comment by syntapy** [comment]
Ok @nbeloglazov, can you merge it? I don't have permissions to do so

**Comment by nbeloglazov** [comment]
Ah, didn't realize you can't merge it. Done.

**Pull request merged**